src/OpenRiaServices.Tools/Test/DomainServiceCatalogTests.cs

```diff
@@ -55,12 +53,16 @@ public void DomainServiceCatalog_Ctors()
             Assert.AreEqual(1, descriptions.Count(), "Expected exactly one domain service description");
 
             // Ctor taking multiple type -- legit
-            dsc = new DomainServiceCatalog(new Type[] {typeof(DSC_DomainServiceType)}, logger);
+            dsc = new DomainServiceCatalog(new Type[] { typeof(DSC_DomainServiceType) }, logger);
             descriptions = dsc.DomainServiceDescriptions;
             Assert.IsNotNull(descriptions, "Did not expect null descriptions");
             Assert.AreEqual(1, descriptions.Count(), "Expected exactly one domain service description");
         }
 
+#if !NETFRAMEWORK
+        [Ignore("Do no work in NET6")]
+        [TestCategory("NET6_ERRORS")]
```

Comment:
We should remove this test category (and try to remove the ignore attribute where possible)